src/index.js

```diff
@@ -75,6 +75,6 @@ export default function supernova(env) {
         [element]
       );
     },
-    ext: ext({ translator, shouldHide, senseNavigation }),
+    ext: ext({ translator, shouldHide, senseNavigation, ext, theme: sense?.theme }),
```

Comment:
You are passing the imported `ext` function as a property here.

Also `env.sense.theme` is deprecated. Use `env.anything.sense.theme` instead.